components/instalation/instalation.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="instalation.css">
+</head>
+<script src="https://kit.fontawesome.com/6b52d0ce66.js" crossorigin="anonymous"></script>
+
+<body>
+    <nav class="header">
+        <div class="name_heading">
+            <h1 id="ui_name">CheckMate-UI</h1>
+            <a class="head_link" href="/index.html">Home</a>
+            <a class="head_link" href="#">Documentation</a>
+        </div>
+        <div class="contact_links">
+            <a class="head_link" href="https://github.com/NamrataRaikwar2002"><i class="fab fa-github"></i></a>
+            <a class="head_link" href="https://www.linkedin.com/in/namrata-raikwar-727951224/"><i class="fab fa-linkedin"></i></a>
+        </div>
+    </nav>
+    <div class="drawer_component">
+        <div class="component_list">
+        <h2>Getting started</h2>
+        <a href="/components/instalation/instalation.html" class="component_item">Installation</a>
+        <h2>Components</h2>
+        <a href="/components/avatar/avatar.html" class="component_item">Avatar</a>
+        <a href="#" class="component_item">Alert</a>
+        <a href="#" class="component_item">Badge</a>   
+        <a href="#" class="component_item">Button</a>
+        <a href="#" class="component_item">Card</a>
+        <a href="#" class="component_item">Image</a>
+        <a href="#" class="component_item">Input</a>
+        <a href="#" class="component_item">Text Utilities</a>
+        <a href="#" class="component_item">List</a>
+        <a href="#" class="component_item">Navigation</a>
+        <a href="#" class="component_item">Modal</a>
+        <a href="#" class="component_item">Rating</a>
+        <a href="#" class="component_item">Snackbar</a>
+        <a href="#" class="component_item">Simplified Grid</a>
+        </div>
+        <section class="installation_text">
+            <h1>Installation</h1>
+            <p>
+            </p>
```

Comment:
Redundant `<p></p>` tag here. Ignore this if you've done that intentionally or else you could fix that.

Reply:
I forgot to remove it, I will remove.